persistence/mongo/src/main/scala/za/co/absa/spline/persistence/mongo/dao/BaselineLineageDAO.scala

```diff
@@ -47,7 +47,9 @@ abstract class BaselineLineageDAO extends VersionedLineageDAO with Logging {
   protected lazy val dataLineageCollection: DBCollection = getMongoCollectionForComponent(Component.Root)
   protected lazy val operationCollection: DBCollection = getMongoCollectionForComponent(Component.Operation)
 
-  private final val NON_WORD_CHAR = "\\W".r
+  object BaselineLineageDao {
```

Comment:
No, I meant place it into the companion object of the `BaselineLineageDAO` class. It already exists at the bottom of the file.

Reply:
Got it. Fixed.